packages/block-library/src/image/image.js

```diff
@@ -1043,10 +1061,43 @@ export default function Image( {
 		);
 	}
 
+	/**
+	 * Set the post's featured image with the current image.
+	 */
+	const setPostFeatureImage = () => {
+		setFeaturedImage( id );
+	};
+
+	/**
+	 * Create a menu item to set the image as the post's featured image.
+	 *
+	 * @param {Object} props                   - Props for block settings menu control.
+	 * @param {Object} props.selectedClientIds - Selected client ids.
+	 *
+	 * @return {void} - Menu item to set the image as the post's featured image.
+	 */
+	const featureImageControl = (
+		<BlockSettingsMenuControls>
+			{ ( { selectedClientIds } ) =>
+				( selectedClientIds.length === 1 &&
+					! isDescendentOfQueryLoop &&
+					postId &&
+					storedFeaturedImage !== id &&
```

Comment:
> Also if the current image is set as a feature image then this control won't be visible.

This conditional statement creates an accessibility issue.

When keyboard users click the button, they lose focus because the button disappears. For now, we can simply remove the condition (`storedFeaturedImage !== id`).